modules/swagger-codegen/src/main/resources/objc/model-body.mustache

```diff
@@ -13,7 +13,11 @@
   }
   return self;
 }
-
+{{#vars}}{{^vendorExtensions.x-unique-id-key}}{{/vendorExtensions.x-unique-id-key}}{{#vendorExtensions.x-unique-id-key}}
+-(NSString*)indexPropertyName {
+    return @"{{name}}";
+}
+{{/vendorExtensions.x-unique-id-key}}{{/vars}}
```

Comment:
@mateuszmackowiak thanks for the explanation. Given that it takes a boolean value, what about naming it as "x-is-unique" instead, which means the value must be unique if set to true?